From: Russ Allbery <eagle@eyrie.org>
Date: Fri, 15 May 2020 18:53:05 -0700
Subject: Allow the environment to override CFLAGS

Do not force CFLAGS in src/Makefile and allow the environment to
override.  This fixes discarding of hardening flags set by the
Debian build system.
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 16365b7..fec80f9 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -1,4 +1,4 @@
-CFLAGS		= -g -Wall -Wsign-compare
+CFLAGS		?= -g -Wall -Wsign-compare
 
 include Makefile.config
 
